src/sage/combinat/matrices/hadamard_matrix.py

```diff
@@ -320,6 +320,87 @@ def hadamard_matrix_paleyII(n):
     return normalise_hadamard(H)
 
 
+def hadamard_matrix_from_symmetric_conference_matrix(n, existence=False, check=True):
+    r"""Construct a Hadamard matrix of order `n` from a symmetric conference matrix
+    of order `n/2`.
+
```

Comment:
```sage    
r"""
Construct a Hadamard matrix of order `n` from a symmetric conference matrix
of order `n/2`.
```
just to conform to the usual style.

Reply:
fixed by [4d56220](https://github.com/sagemath/sage/pull/38486/commits/4d562209ed2488da9ac60604a9ff48324218da18)

Reply:
Thanks.